macros/base/start_print.cfg

```diff
@@ -349,9 +349,11 @@ gcode:
     {% if ercf_enabled %}
         {% if ercf_reset_stats_on_start_print %}
             ERCF_RESET_STATS
-        {% endif %}            
-        ERCF_CHANGE_TOOL STANDALONE=1 TOOL={INITIAL_TOOL}
-        ### _DO_ERROR_ERCF_PAUSE
+        {% endif %}
+        {% if printer.ercf.tool|int != INITIAL_TOOL or printer.ercf.filament != "Loaded" %}
```

Comment:
This is done automatically by HH. why would you want to repeat its internal function?